README.md

```diff
@@ -215,6 +215,37 @@ The possible values for the annotation can be
 * `"my-instrumentation"` - name of `Instrumentation` CR instance.
 * `"false"` - do not inject
 
+It as also possible in case of multiple-containers Pods (like Istio sidecars) to specify the name of the container in which you want to inject OTEL auto-instrumentation (by default the first container is choosen)
+
+For this add this annotation to your Pod Specification with target container name :
+
+```bash
+instrumentation.opentelemetry.io/container-names: "name_of_your_target_container"
+```
+In case of injection on multiple containers, you can indicates all your containers'name separate with comma.
+
+#### Use customized or vendor instrumentation
```

Comment:
This change should not be here, maybe the PR was not rebased properly?

Reply:
This section should not be added in this PR

Reply:
OK. I didn't understand you don't want any modifications in documentation. 
I remove all of this and get Readme from main branch of main repo.